src/control/server/config_test.go

```diff
@@ -372,28 +390,31 @@ func TestSetNumCores(t *testing.T) {
 			ExpectError(t, err, tt.errMsg, tt.num)
 			continue
 		}
+
 		AssertEqual(t, cpus, tt.cpus, "failed to convert number to range")
+
 		num, err := getNumCores(cpus)
 		if err != nil {
 			t.Fatal(err)
 		}
+
 		AssertEqual(t, num, tt.num, "failed to convert to expected number")
 	}
 }
 
 // TestCmdlineOverride verified that cliOpts take precedence over existing
 // configs resulting in overrides appearing in ioparams
 func TestCmdlineOverride(t *testing.T) {
-	r := rank(9)
+	//r := rank(9)
```

Comment:
Did you mean to leave these lines commented out?

Reply:
no, thanks for spotting